test/powershell/Modules/PowerShellGet/PowerShellGet.Tests.ps1

```diff
@@ -197,17 +197,17 @@ Describe "PowerShellGet - Script tests" -tags "Feature" {
 
     It "Should find a script correctly" {
         $psgetScriptInfo = Find-Script -Name $FabrikamServerScript -Repository $RepositoryName
-        $psgetScriptInfo.Name | Should Be $FabrikamServerScript
-        $psgetScriptInfo.Repository | Should Be $RepositoryName
+        $psgetScriptInfo.Name | Should -Be $FabrikamServerScript
+        $psgetScriptInfo.Repository | Should -Be $RepositoryName
     }
 
     It "Should install a script correctly to the required location with CurrentUser scope" {
         Install-Script -Name $FabrikamServerScript -Repository $RepositoryName -Scope CurrentUser -NoPathUpdate
         $installedScriptInfo = Get-InstalledScript -Name $FabrikamServerScript
 
-        $installedScriptInfo | Should Not Be $null
-        $installedScriptInfo.Name | Should Be $FabrikamServerScript
-        $installedScriptInfo.InstalledLocation.StartsWith($script:MyDocumentsScriptsPath, [System.StringComparison]::OrdinalIgnoreCase) | Should Be $true
+        $installedScriptInfo | Should -Not -BeNullOrEmpty
```

Comment:
I think we should keep Should -Not -Be $null because `$installedScriptInfo` is an object not string.

Reply:
Per Pester `-BeNullOrEmpty` is applied to both objects and strings, I think it's fine to use it here.